index.html

```diff
@@ -28,6 +28,7 @@ <h1>Dev Delta Jokes</h1>
         <li><a href="sam-mcknight/index.html">Sam's Joke Page</a></li>
         <li><a href="sean/joke.html">Sean's Joke Page</a></li> 
         <li>Shannon Placeholder</li>
+        <li><a href="../dev-delta-jokes/Jonah/index.html">Jonah's Joke Page</a></li>
```

Comment:
you should be able to just do 
```suggestion
        <li><a href="Jonah/index.html">Jonah's Joke Page</a></li>
```